src/libstd/collections/hash/state.rs

```diff
@@ -51,5 +51,6 @@ impl<H> Clone for DefaultState<H> {
 }
 
 impl<H> Default for DefaultState<H> {
+    /// Creates a new `DefaultState` containing `marker::PhantomData`
```

Comment:
`PhantomData` is an implementation detail of `DefaultState`. It shouldn't be documented.